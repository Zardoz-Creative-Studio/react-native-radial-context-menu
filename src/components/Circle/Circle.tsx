import * as React from "react";
import { View } from "react-native";
import { styles } from "../../styles";

export interface ICircleProps {
  style: object;
}

// tslint:disable-next-line:typedef
export const Circle: React.SFC<ICircleProps> = ({ style }: ICircleProps) => (
  <View style={[styles.circle, style]} />
);
