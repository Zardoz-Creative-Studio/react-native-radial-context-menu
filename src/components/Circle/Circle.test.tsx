import * as React from "react";
import renderer from "react-test-renderer";
import { Circle, ICircleProps } from "./Circle";

describe("Circle", () => {
  it("renders the component correctly", () => {
    const props: ICircleProps = {
      style: {},
    };

    expect(renderer.create(<Circle {...props} />).toJSON()).toMatchSnapshot();
  });
});
