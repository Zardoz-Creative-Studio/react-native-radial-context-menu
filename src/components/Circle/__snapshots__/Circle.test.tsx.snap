// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Circle renders the component correctly 1`] = `
<View
  style={
    Array [
      Object {
        "borderColor": "green",
        "borderRadius": 999,
        "position": "absolute",
        "zIndex": 999,
      },
      Object {},
    ]
  }
/>
`;
